int64 ID
int64 label
geometry_msgs/Pose Pose
int64 drillOp
int64 reach
color color